pages/blog/asyncapi-discovery-intro.md

```diff
@@ -0,0 +1,75 @@
+---
+type: Engineering
+cover: /img/posts/async_standards_compare/devops-specs_pic-01.webp
+title: "Align Production Reality and Event Documentation with the AsyncAPI Discovery Tool"
+date: 2021-11-24T06:00:00+01:00
+tags:
+   - Discovery
+   - Specification
+   - EDA
+   - Kafka
+   - Solace
+   - RabbitMQ
+   - NATS
+authors:
+   - name: Jesse Menning
+     photo: /img/avatars/jmenning.webp
+     link: https://linkedin.com/in/jmenning
+     byline: CTO Architect at Solace
+---
+
+The idealized world of AsyncAPI is neat and tidy:
+
+- Interface definitions show developers exactly what events are exchanged amongst producers and consumers.
+- Event contracts only change when permitted by well-defined governance processes.
+- Bindings fill in the implementation details for open-source brokers like Kafka (including Confluent) and RabbitMQ along with closed-source brokers like IBM MQ and Solace.
+
+The real world is often messier. 
+
+> This post [Align Production Reality and Event Documentation with the AsyncAPI Discovery Tool](https://solace.com/blog/asyncapi-discovery-tool/) appeared first on [Solace](https://solace.com).
+
+In the decades-long absence of an asynchronous API spec, teams adopted (or didn’t adopt, or chose to ignore) different API definitions and governance. The result is a twisted mess of event producers, consumers, data paths, and multiple broker technologies, from on-premises to cloud connections.
+
+Oftentimes, removing an event topic or queue requires nerves of steel, for fear it might disrupt key functionality. Many a middleware engineer has found religion during a production broker clean-up.
+
+Thankfully, the AsyncAPI Discovery Tool offers a better way.
+# How the AsyncAPI Discovery Tool Works
+The AsyncAPI Discovery Tool analyzes event traffic passing through brokers like Kafka, RabbitMQ, IBM MQ, Solace, and more. After learning how the broker distributes events, the AsyncAPI Discovery Tool generates a corresponding [AsyncAPI specification](https://www.asyncapi.com/docs/specifications/v2.2.0). The generated spec can be used for code generation, documentation, visualization, infrastructure deployment, and more.
```

Comment:
```suggestion
The AsyncAPI Discovery Tool analyzes event traffic passing through brokers like Kafka, RabbitMQ, IBM MQ, Solace, and more. After learning how the broker distributes events, the AsyncAPI Discovery Tool generates a corresponding [AsyncAPI specification](https://www.asyncapi.com/docs/specifications/latest). The generated spec can be used for code generation, documentation, visualization, infrastructure deployment, and more.
```

Using `latest` as your last parameter instead of the hardcoded `v.2.2.0` will automatically redirect to the latest spec.